### DISCUSSION TITLE
Where can I find KeepPassXC unix domain socket on macOS?

### DISCUSSION THREAD

**Discussion by sebastianlivoni** [keepassxreboot/keepassxc#10967] [Q&A] [answered]
Can't seem to locate the socket on macOS named org.keepassxc.KeePassXC.BrowserServer. Can anybody provide me with an URL for it?

I can see that the path is:
`return QStandardPaths::writableLocation(QStandardPaths::TempLocation) + serverName;`
from https://github.com/keepassxreboot/keepassxc/blob/089c8df01d1676abb6d084edf4da7038bc0d9be2/src/browser/BrowserShared.cpp#L34 but I can't find any file or folder named org.keepassxc.KeePassXC.BrowserServer

**Comment by varjolintu** [comment, marked as answer]
`echo $TMPDIR` will show you the current user's temporary path where the socket file will be found.
